ui/src/app/utils/resourceActions.ts

```diff
@@ -41,4 +54,40 @@ export const cohortActionList = [
 
 export const resourceActionList =  notebookActionList.concat(cohortActionList);
 
+export function convertToResources(list: any[], component): RecentResource[] {
```

Comment:
I'm guessing you might sort this list later in the components, but this function would be better if it preserved the order of the input list